Packs/Thycotic/Integrations/Thycotic/Thycotic.yml

```diff
@@ -0,0 +1,1096 @@
+      description: Fields to search
+      isArray: false
+      name: filter_searchfields
+      required: false
+      secret: false
+    - default: false
+      description: Search text
+      isArray: false
+      name: filter_searchtext
+      required: false
+      secret: false
+    - default: false
+      description: Number of records to skip before taking results
+      isArray: false
+      name: skip
+      required: false
+      secret: false
+    - default: false
+      description: Sort direction
+      isArray: false
+      name: sortBy[0]_direction
+      required: false
+      secret: false
+    - default: false
+      description: Sort field name
+      isArray: false
+      name: sortBy[0]_name
+      required: false
+      secret: false
+    - default: false
+      description: Priority index. Sorts with lower values are executed earlier (integer)
+      isArray: false
+      name: sortBy[0]_priority
+      required: false
+      secret: false
+    - default: false
+      description: Maximum number of records to include in results(integer)
+      isArray: false
+      name: take
+      required: false
+      secret: false
+    deprecated: false
+    description: Search, filter, sort, and page users
+    execution: false
+    name: thycotic-user-search
+    outputs:
+    - contextPath: Thycotic.User.Search
+      description: List of User IDs
+      type: String
+  - arguments:
+    - default: false
+      description: User ID
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    - default: false
+      description: DateOptionId(integer)
+      isArray: false
+      name: dateoptionid
+      required: false
+      secret: false
+    - default: false
+      description: Display name
+      isArray: false
+      name: displayname
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether Duo two-factor authentication is enabled
+      isArray: false
+      name: duotwofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: E-mail
+      isArray: false
+      name: emailaddress
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether the user account is enabled
+      isArray: false
+      name: enabled
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether FIDO2 two-factor authentication is enabled
+      isArray: false
+      name: fido2twofactor
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: GroupOwners(integer)
+      isArray: false
+      name: groupowners
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: IsApplicationAccount
```

Comment:
Many of the arguments' descriptions need to be replaced.
Some examples can be found below (I'll add them as notes), but please review all of them and replace them with meaningful, detailed descriptions

Reply:
@guykeller 
All descriptions of commands and arguments are taken from the basic API documentation.
I myself have not used many and do not know their correct application.

Reply:
IsApplicationAccount, true - if this is account usage for SDK

Reply:
I don't understand what you mean by that, can you elaborate?

Reply:
@guykeller 
I have requested a more detailed explanation of the API parameters.
In this case, a service account is used to connect to the SecretServer using the SDK